src/ui/public/agg_types/metrics/lib/sibling_pipeline_agg_controller.js

```diff
@@ -0,0 +1,31 @@
+import safeMakeLabel from './safe_make_label';
+
+const siblingPipelineAggController = function (type) {
+  return function ($scope) {
+
+    $scope.aggType = type;
+    //$scope.aggParam = $scope.agg.params[type];
+    $scope.aggTitle = type === 'customMetric' ? 'Metric' : 'Bucket';
+    $scope.aggGroup = type === 'customMetric' ? 'metrics' : 'buckets';
```

Comment:
Looks like there is `aggGroup` and also `agg.group` and they mean different things.  I'm not sure I have a suggestion on the naming, but it'd be nice if they could be distinguished better.

Almost seems like maybe typeName would be more fitting?  I think I tracked this down to being used here:
`aggTypes.byType[$scope.groupName];`

Though that might be too far reaching to change it in this PR since that name already exists and group is the new name.

I think something else that would be worthwhile as a separate PR would be moving `metrics` and `buckets` to constants too.  That would be easier to understand the importance of these strings and where else they are being referenced.  Then you could do something like:
```
export const AggTypes = {
  METRICS: 'metrics',
  BUCKETS: 'buckets'
}
```

Food for thought mostly, I understand if you prefer to leave as is for lack of a better alternative at this point.